### PR TITLE
Add react-dom and utility bind method

### PR DESCRIPTION
**Pull request by asvny**
Good old days like Backbone.js  _.bindAll :wink:

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -35,6 +39,8 @@ export class FloatingLabel extends React.Component {
 
     const inputClasses = classNames('fl-input', {'fl-valid': hasValue && !hasError}, {'fl-invalid': hasValue && hasError});
     const errMsgClasses = classNames({'fl-error-msg': errorMsg}, {'fl-error-show': (hasError && hasValue) && (errorMsg && pattern)});
+    
+    console.log(inputClasses);
```
Comment (hemanth) [inline main.js]: Please remove this and squash the commits.

File: main.js [hunk 2]
```diff
@@ -11,10 +12,13 @@ export class FloatingLabel extends React.Component {
     id: PropTypes.string.isRequired,
     isDisabled: PropTypes.bool
   };
-
+  _bind(...methods) {
+    methods.map((method)=>this[method]=this[method].bind(this));
```
Comment (hemanth) [inline main.js]: Just `method` is fine, we don't need the parans.

**Comment by hemanth** [comment]
Thanks, I am still OK with old school binding.

//cc @gokulkrishh

**Comment by annamalai-saro** [comment]
good to go .. I guess !

**Comment by hemanth** [comment]
@annamalai-saro Sorry, could you rebase this?

**Pull request merged**